substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/util/ExitStatus.java

```diff
@@ -33,15 +33,16 @@ public enum ExitStatus {
     DRIVER_ERROR(20),
     DRIVER_TO_BUILDER_ERROR(21),
     WATCHDOG_EXIT(30),
-    MISSING_METADATA(172);
+    MISSING_METADATA(172),
+    UNKNOWN(255);
 
     public static ExitStatus of(int status) {
         for (ExitStatus s : values()) {
             if (s.getValue() == status) {
                 return s;
             }
         }
-        return null;
+        return UNKNOWN;
```

Comment:
Exit code 137 is common and important - maybe you should have a constant for it. Returning `UNKNOWN` may hide the actual reason why it is happening (out of memo ry of Linux system) and may make it hard to understand how that can be fixed (close Chrome & Firefox & co. or set a swap space).

Reply:
At least I was [quite desperate](https://graalvm.slack.com/archives/CN9KSFB40/p1660539227687399) when I got 137 for the first time.

Reply:
This PR will fix the NPE. Also, this method only has one caller and there, the exit code will still be printed.
Anyway, I'm also looking into a better error message in case of OOMEs, although as part of another PR.

Reply:
See https://github.com/oracle/graal/pull/5782